main.go

```diff
@@ -271,28 +326,58 @@ func (z *Config) Archive(meeting zoom.Meeting) error {
 	z.logger.Printf("archiving meeting %d to %s (https://drive.google.com/drive/folders/%s)",
 		meeting.ID, meetingFolder.Name, meetingFolder.Id)
 	for _, f := range meeting.RecordingFiles {
+
+		//check if meeting duration is shorter than minimum
+		start, err := time.Parse(time.RFC3339, f.RecordingStart)
+		if err != nil {
+			curArchMeeting.status = "error"
+			return fmt.Errorf("while parsing recording start %s - %s: %w", f.ID, f.RecordingStart, err)
```

Comment:
should we just log the error but continue to uploading the recording in this case?

Reply:
Yes, that, was my intention, hence the if down below. Let me replace that return by a log.